### PR TITLE
Update FAQs to reflect recent changes.

### PR DESCRIPTION
**Pull request by MissPotato**
I'm uncertain what other gameplay changes are made but felt like this is a reasonable start to help.

### CODE REVIEW

**Comment by MissPotato** [comment]
Could you clarify "Bed respawn/dismount mechanic changes for players" are you talking about preventing players from getting kicked for "flying" while actually sleeping or is there other changes?

And I think sync should be explained by someone more knowledgable about the changes than my self.

**Comment by kickash32** [comment]
Here are the bed changes https://github.com/PaperMC/Paper/issues/1726 (villagers are unaffected).

As for the sync chunk loading changes, I'm not sure myself.

File: source/about/faq.rst [hunk 1]
```diff
@@ -50,12 +50,29 @@ to see a noticeable performance improvement.
...
 
+You may also notice changes to gameplay. 
+
 Will players be able to tell?
 -----------------------------
 
 That depends. Your players may see a benefit to gameplay because of the
 performance improvement, assuming you see one. On a properly maintained server,
-your players may not even be able to tell the difference.
+your players may not even be able to tell the difference unless it is content we
```
Comment (electronicboy) [inline source/about/faq.rst:60]: This just sounds like somebody knocked over a pot of salt

Comment (MissPotato) [reply source/about/faq.rst:60]: It could use some rewording. It sounds pretty antagonistic in retrospect.

**Comment by MissPotato** [comment]
Could include a hyperlink to the new page in the FAQs. I will edit out the that section and go through rest of this with Grammarly.

**Comment by zachbr** [comment]
Closing due to inactivity.

If someone else wants to pick this up, please see Aikar's comment on making a separate page for this.
https://github.com/PaperMC/PaperDocs/pull/33#pullrequestreview-455296732

Thanks

**Pull request closed without merging**